gallery/tutorials/aspire_introduction.py

```diff
@@ -0,0 +1,675 @@
+# other Computational CryoEM data simulation tasks.
+
+# %%
+# The ``filter`` Package
+# ----------------------
+# `Filters
+# <https://computationalcryoem.github.io/ASPIRE-Python/aspire.operators.html#module-aspire.operators.filters>`_
+# are a collection of classes which once configured can be applied to
+# ``Images``, typically in an ``ImageSource`` pipeline which will be
+# discussed in a later section.
+
+# %%
+#
+#  .. mermaid::
+#
+#    classDiagram
+#        class Filter{
+#            +evaluate()
+#            +fb_mat()
+#            +scale()
+#            +evaluate_grid()
+#            +dual()
+#            +sign()
+#         }
+#
+#         Filter o-- DualFilter
+#         Filter o-- FunctionFilter
+#         Filter o-- PowerFilter
+#         Filter o-- LambdaFilter
+#         Filter o-- MultiplicativeFilter
+#         Filter o-- ScaledFilter
+#         Filter o-- ArrayFilter
+#         Filter o-- ScalarFilter
+#         Filter o-- ZeroFilter
+#         Filter o-- IdentityFilter
+#         Filter o-- CTFFilter
+#         CTFFilter o-- RadialCTFFilter
+
+# %%
+# ``CTFFilter`` and ``RadialCTFFilter`` are the most common filters
+# encountered when starting out and are detailed in
+# :ref:`sphx_glr_auto_tutorials_tutorials_ctf.py` The other filters
+# are used behind the scenes in components like ``NoiseAdders`` or
+# more advanced customized pipelines.
+
+# %%
+# ``Basis``
+# ---------
+# ASPIRE provides a selection of ``Basis`` classes designed for
+# working with CryoEM data in two and three dimensions.  Most of these
+# basis implementations are optimized for efficient rotations, often
+# called the *"Steerable"* property.  As of writing most algorithms in
+# ASPIRE are written to work well with the Fast Fourier Bessel Basis
+# classes ``FFBBasis2D`` and ``FFBBasis3D``.  These correspond to
+# direct slower reference ``FBBasis2D`` and ``FBBasis3D`` methods.
+#
+# Recently, a related Fourier Bessel method using Fast Laplacian
+# Eigenfunction transforms was integrated as ``FLEBasis2D``.
+# Additional Prolate Spheroidal Wave Function methods are available
+# via ``FPSWFBasis2D`` and ``FPSWFBasis3D``, but their integration
+# into other components like 2D covariance analysis is incomplete, and
+# slated for a future release.
+
+# %%
+# The ``source`` Package
+# ----------------------
+#
+# `aspire.source
+# <https://computationalcryoem.github.io/ASPIRE-Python/aspire.source.html#module-aspire.source.simulation>`_
+# package contains a collection of data source interfaces.
+# Ostensibly, a ``Source`` is a producer of some primitive type, most
+# notably ``Image``.  ASPIRE components that consume (process) images
+# are designed to accept an ``ImageSource``.
+#
+# The first reason for this is to normalize the way a wide variety of
+# higher level components interface.  ``ImageSource`` instances have a
+# consistent method ``images`` which must be implemented to serve up
```

Comment:
Well it's presented as an array attribute (or property), no (with the `ImageAccessor` magic)?

Reply:
Yeah, technically it is a property.  I'm not sure we want to pull the curtain off exactly how we're achieving that syntax to new users (in case it changes). 

Updated method to property.  I avoided the stuff that should be opaque while still trying to give some insight.

```
``ImageSource`` instances have a                                                                    
# consistent property ``images`` which must be implemented to serve up                                                                    
# images dynamically using a square-bracket ``[]`` syntax familiar to                                                                     
# Numpy users. 
```

Reply:
Great. Thanks!